From cb6a7cc2bd7bb361c313a785eb76edc60de7bdce Mon Sep 17 00:00:00 2001
From: Shivank Sharma <shivanksharma2376543@gmail.com>
Date: Thu, 16 Jul 2026 21:39:44 +0530
Subject: [PATCH 09135/15333] initramfs: fix typo in reserve_initrd_mem comment

Fix a minor typo in the comment inside reserve_initrd_mem.
Change "virtul" to "virtual".

Signed-off-by: Shivank Sharma <shivanksharma2376543@gmail.com>
Link: https://patch.msgid.link/20260716160944.1331096-1-shivanksharma2376543@gmail.com
Signed-off-by: Christian Brauner (Amutable) <brauner@kernel.org>
---
 init/initramfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/initramfs.c b/init/initramfs.c
index 20a18fcda48e..55c17c8f3991 100644
--- a/init/initramfs.c
+++ b/init/initramfs.c
@@ -618,7 +618,7 @@ void __init reserve_initrd_mem(void)
 	phys_addr_t start;
 	unsigned long size;
 
-	/* Ignore the virtul address computed during device tree parsing */
+	/* Ignore the virtual address computed during device tree parsing */
 	initrd_start = initrd_end = 0;
 
 	if (!phys_initrd_size)
-- 
2.55.0

